src/lua/LuaPlayer.cpp

```diff
@@ -361,6 +351,12 @@ static int l_get_remaining_delta_v(lua_State *l)
 	return 1;
 }
 
+static std::map<std::string, Thruster> thrusters_map = {
+	{ "forward", THRUSTER_FORWARD },
+	{ "reverse", THRUSTER_REVERSE },
+	{ "up", THRUSTER_UP },
+};
+
```

Comment:
If this map isn't used in a new way that requires moving it within the source file, please just leave it in-place - it makes it easier to follow the git history / blame log.